lib/sqlitex/row.ex

```diff
@@ -56,8 +56,10 @@ defmodule Sqlitex.Row do
     {String.to_integer(yr), String.to_integer(mo), String.to_integer(da)}
   end
 
-  defp to_time(time) do
-    <<hr::binary-size(2), ":", mi::binary-size(2), ":", se::binary-size(2), ".", fr::binary-size(6)>> = time
+  defp to_time(<<hr::binary-size(2), ":", mi::binary-size(2), ":", se::binary-size(2), ".", fr::binary-size(6)>>) do
```

Comment:
when we translate that to a 4-tuple, isn't there an assumption that the last integer represents a number of microseconds? So we would need to convert from milliseconds to microseconds?